### ISSUE TITLE
broken with stylelint 16.10: TypeError: Cannot read properties of undefined (reading 'config')

### ISSUE THREAD

**Issue by MiniDigger** [merkle-open/stylelint-bem#32]
## Type of issue

* Bug report

## Environment

```
Node.js v22.2.0
win32 10.0.19045
```

## Expected behavior

stuff works

## Current behavior

```
    TypeError: Cannot read properties of undefined (reading 'config')
        at Object.validateOptions (C:\Gothaer\Projects\stylelint-bem\node_modules\stylelint\lib\utils\validateOptions.cjs:22:40)
        at C:\Gothaer\Projects\stylelint-bem\index.js:181:40
        at file:///C:/Gothaer/Projects/stylelint-bem/node_modules/stylelint/lib/lintPostcssResult.mjs:113:60
        at Array.map (<anonymous>)
        at lintPostcssResult (file:///C:/Gothaer/Projects/stylelint-bem/node_modules/stylelint/lib/lintPostcssResult.mjs:112:18)
        at async lintSource (file:///C:/Gothaer/Projects/stylelint-bem/node_modules/stylelint/lib/lintSource.mjs:108:2)
        at async standalone (file:///C:/Gothaer/Projects/stylelint-bem/node_modules/stylelint/lib/standalone.mjs:143:26)
        at async C:\Gothaer\Projects\stylelint-bem\node_modules\jest-preset-stylelint\getTestRule.js:73:30
```

## Steps to reproduce the behavior

* clone the repo
* update stylelint to 16.10
* run the tests

## Explanation

so this is a fun one. I thought I was going insane for a bit, but managed to track it down.
this package is calling stylelint's validateOptions with an invalid method signature!
it passes an object: https://github.com/merkle-open/stylelint-bem/blob/41ee8429b961d730c1ccea752a6fc9b8ed6ce315/index.js#L181-L185
when stylelint expects multiple params: https://github.com/stylelint/stylelint/blob/1b03662ecb05cc6ed323b4e3b0946fb20870b287/lib/utils/validateOptions.cjs#L20-L22

this never surfaced, since validateOptions basically does nothing if the 3rd param is empty. At least it used to do nothing. in stylelint 16.10 there is a new option that actually checks stuff on the first param, and since stylelint-bem passes a malformed param, the error occurs. commit adding the option: https://github.com/stylelint/stylelint/commit/86437547861e164028cb23b3feec965c7816de00#diff-a9078006ea40b10e08ed8dd13f68bb5a9585641b76e827cef5bc80216d1aa877R18 )

ill submit a PR fixing it after lunch....

**Comment by ernscht** [comment]
Great!
Thank you very much for your finding and your work.

But I won't be able to look at it until tomorrow at the earliest.

**Comment by ernscht** [comment]
Fix is [published as v10.1.0](https://www.npmjs.com/package/@namics/stylelint-bem)
